src/main/java/Duke.java

```diff
@@ -1,10 +1,125 @@
+import  java.util.Scanner;
+
 public class Duke {
+
+    static final int MAX_NO_OF_TASKS = 100;
+
     public static void main(String[] args) {
+        showWelcomeMessage();
+
+        Task[] userTasks = new Task[MAX_NO_OF_TASKS];
+        int taskCounter = 0;
+        boolean exitLoopStatus = false;
+        Scanner inputCommand = new Scanner(System.in);
+        String userInput;
+
+        while(true){
```

Comment:
Should this while loop be extracted out? I think it might be cleaner if abstracted